Libraries/src/Amazon.Lambda.RuntimeSupport/Client/RuntimeApiClient.cs

```diff
@@ -92,10 +92,11 @@ public Task ReportInitializationErrorAsync(string errorType)
         /// Get the next function invocation from the Runtime API as an asynchronous operation.
         /// Completes when the next invocation is received.
         /// </summary>
+        /// <param name="cancellationToken">The optional cancellation token to use to stop listening for the next invocation.</param>
         /// <returns>A Task representing the asynchronous operation.</returns>
-        public async Task<InvocationRequest> GetNextInvocationAsync()
+        public async Task<InvocationRequest> GetNextInvocationAsync(CancellationToken cancellationToken)
```

Comment:
I'm okay with this slight breaking change but we should make it `cancellationToken = default` so it won't break users that recompile their application.

Also we might as well add cancellationToken to all of the methods in IRuntimeApiClient to to make the one "breaking" change all at once. 

I don't really anticipate users calling IRuntimeApiClient directly which is why I'm okay making the breaking change.